roles/foreman_installer/tasks/trust_github.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: 'Add SSH_AUTH_SOCK to env_keep in sudoers'
+  lineinfile:
+    dest: /etc/sudoers
+    state: present
+    regexp: SSH_AUTH_SOCK
+    line: 'Defaults env_keep += "SSH_AUTH_SOCK"'
```

Comment:
So... I think there might be a better way to do this, but since we're using `become: true` in Ansible, the ssh agent forwarding doesn't work without this environment variable. See [this issue](https://github.com/ansible/ansible/issues/7235) in Ansible.

Reply:
Another approach is more of a git trick in ~vagrant/.gitconfig
```
[url "git@github.com:"]
        pushInsteadOf = "https://github.com/"
```

This can be set by ```git config --global url."git@github.com:".pushInsteadOf https://github.com/```

Pulling over HTTPS is fine since they're public repos and the installer doesn't need to push.

Reply:
So, this would mean I would disable the ssh fork and set this setting to push only over SSH, right?

Reply:
Correct.